core/src/main/scala/org/apache/spark/h2o/H2OContext.scala

```diff
@@ -255,7 +265,142 @@ object H2OContext extends Logging{
     * @return H2O Context
     */
   def getOrCreate(sc: SparkContext): H2OContext = {
-    getOrCreate(sc, new H2OConf(sc))
+    getOrCreate(sc, new H2OConf(sc))(SQLContext.getOrCreate(sc))
+  }
+
+  val defaultFieldNames = (i: Int) => "f" + i
+
+  def buildH2OFrame(kn: String, vecTypes: Array[Byte], res: Array[Long]): H2OFrame = {
```

Comment:
This function seems like internal function which should not be visible on H2OContext, therefore I wouldn't put it in H2OContext and `ConvertorUtils` should stay private to convertors.

Maybe better than defining this function would be to use `water.fvec.Frame` constructor 
`public Frame( Key key, String names[], Vec vecs[] )` in places you need to create a frame.